Production/plugins/TupleProducer_muMu.cc

```diff
@@ -72,10 +71,13 @@ void TupleProducer_muMu::ProcessEvent(Cutter& cut)
     if(runSVfit)
         selection.svfitResult.push_back(svfitProducer->Fit(selected_higgs, *met));
 
-    selection.other_muons = CollectVetoMuons({ &selected_higgs.GetFirstDaughter(),
+    selection.other_muons = CollectVetoMuons(false,{ &selected_higgs.GetFirstDaughter(),
         &selected_higgs.GetSecondDaughter() });
     selection.muonVeto = selection.other_muons.size();
-    cut(!selection.muonVeto, "no_extra_muon");
+
+    selection.other_tight_muons = CollectVetoMuons(true,{ &selected_higgs.GetFirstDaughter(),
+        &selected_higgs.GetSecondDaughter() });
+    cut(selection.other_tight_muons.size() == 0, "no_extra_muon");
```

Comment:
here and in other places check that tight cuts are applied as soon as possible to reduce computation time. For example, in this case it should be applied definitely before running svfit.